spec/models/runs_extractors_spec.rb

```diff
@@ -1,3 +1,5 @@
+class DummyException < StandardError; end
```

Comment:
I'd scope this error definition to where it's being used so i can see it quickly, i.e. under `describe 'error handling' do` block def.